src/simulation/power/simpleBattery/simpleBattery.rst

```diff
@@ -36,3 +36,8 @@ The next step is to attach one or more :ref:`PowerNodeUsageMsgPayload` instances
 
 
 For more information on how to set up and use this module, see the simple power system example :ref:`scenarioPowerDemo`.
+
+To simulate a battery capacity fault that reduces the actual storage capacity (without directly altering the stated capacity), users must set up the fault flag::
+
+   battery.fault = True
```

Comment:
This should be updated to use the private class setter function.  Check out 
https://avslab.github.io/basilisk/Documentation/moduleTemplates/cppModuleTemplate/cppModuleTemplate.html
to see how to name and document the setter and getter functions.